PurchasesTests/Networking/BackendTests.swift

```diff
@@ -1385,11 +1381,8 @@ class BackendTests: XCTestCase {
             completionCalled += 1
         })
 
-        let discount = RCPromotionalOffer()
-        discount.offerIdentifier = "offerid"
-        discount.paymentMode = .payAsYouGo
-        discount.price = 12
-
+        _ = PromotionalOffer(offerIdentifier: "offerid", price: 12, paymentMode: .payAsYouGo)
+        
```

Comment:
was this discount not being used? we should try to figure out why we actually had it, and if it was indeed just a copy/paste thing, just kill it

Reply:
it seems like the original intention was to actually create a `ProductInfo` instance and send it in `postReceiptData`?

Reply:
Yeah, I'm not sure what the intent was, just that it wasn't doing anything, ha. I'll look at that.

Reply:
Updated ✅
Created a `ProductInfo` and sent it along.